halo2_gadgets/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "halo2_gadgets"
+version = "0.0.0"
+authors = [
+    "Sean Bowe <sean@electriccoin.co>",
+    "Jack Grigg <jack@electriccoin.co>",
+    "Daira Hopwood <daira@jacaranda.org>",
+    "Ying Tong Lai <yingtong@electriccoin.co>",
+    "Kris Nuttycombe <kris@electriccoin.co>",
+]
+edition = "2018"
+description = "[BETA] Reusable gadgets and chip implementations for Halo 2"
+license-file = "../COPYING"
+repository = "https://github.com/zcash/halo2"
+readme = "README.md"
+categories = ["cryptography"]
+keywords = ["zcash"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs", "--html-in-header", "../katex-header.html"]
+
+[dependencies]
+arrayvec = "0.7.0"
+bigint = "4"
+bitvec = "0.22"
+ff = "0.11"
+group = "0.11"
+halo2_proofs = { version = "=0.1.0-beta.1", path = "../halo2_proofs" }
+lazy_static = "1"
+pasta_curves = "0.3"
+proptest = { version = "1.0.0", optional = true }
+rand = "0.8"
+subtle = "2.3"
+
+# Developer tooling dependencies
+plotters = { version = "0.3.0", optional = true }
+
+[dev-dependencies]
+criterion = "0.3"
+proptest = "1.0.0"
+
+[target.'cfg(unix)'.dev-dependencies]
+pprof = { version = "=0.6.1", features = ["criterion", "flamegraph"] }
+
+[lib]
+bench = false
+
+[features]
+dev-graph = ["halo2_proofs/dev-graph", "plotters"]
+test-dependencies = ["proptest"]
```

Comment:
Are we using `test-dependencies` anywhere?

Reply:
Nope, it was left over from `orchard`. I've removed it.